src/citrine/ara/variables.py

```diff
@@ -9,6 +10,13 @@
 from citrine._serialization import properties
 
 
+class IngredientQuantityDimension:
```

Comment:
Do these serialize to their string names, or to their numbers?  In `taurus`, we define enumerations with this pattern: https://github.com/CitrineInformatics/taurus/blob/develop/taurus/enumeration/origin.py

Reply:
Since you wrote the code this might be a rhetorical question, but it looks to me like they serialize to strings: https://github.com/CitrineInformatics/platform-backend/blob/develop/code/ara/model/src/main/scala/io/citrine/ara/model/variable/IngredientQuantity.java

Reply:
Yeah, the backend is expecting them as strings.